host/tests/L3_TestCases/dsFPD/dsFPD_L3_testSetup.yml

```diff
@@ -6,15 +6,7 @@ dsFPD:  # Prefix must always exist
   assets:
     device:
       defaults: &defaults
-        artifacts:
-          - "<PATH>/bin/hal_test"
-          - "<PATH>/bin/libut_control.so"
-          - "<PATH>/profiles/source/Source_FPD.yaml"
-          - "<PATH>/profiles/sink/Sink_FPD.yaml"
-          - "<PATH>/bin/run.sh"
         execute:
```

Comment:
once testRunPrerequisites is removed, default section also is not required

Reply:
Removed default section as well.